tests/kafkatest/services/kafka/kafka.py

```diff
@@ -75,11 +93,20 @@ class KafkaService(KafkaPathResolverMixin, JmxMixin, Service):
     def __init__(self, context, num_nodes, zk, security_protocol=SecurityConfig.PLAINTEXT, interbroker_security_protocol=SecurityConfig.PLAINTEXT,
                  client_sasl_mechanism=SecurityConfig.SASL_MECHANISM_GSSAPI, interbroker_sasl_mechanism=SecurityConfig.SASL_MECHANISM_GSSAPI,
                  authorizer_class_name=None, topics=None, version=DEV_BRANCH, jmx_object_names=None,
-                 jmx_attributes=None, zk_connect_timeout=5000, zk_session_timeout=6000, server_prop_overides=None, zk_chroot=None):
+                 jmx_attributes=None, zk_connect_timeout=5000, zk_session_timeout=6000, server_prop_overides=None, zk_chroot=None,
+                 use_separate_interbroker_listener=False):
         """
         :type context
         :type zk: ZookeeperService
         :type topics: dict
+        :param use_separate_interbroker_listener - if set, will use a separate interbroker listener,
```

Comment:
though awkward, let's try to conform to the existing docstring format above

Reply:
hmm, reading the docs, seems like `:type` notation specifies parameter type while `:param` specifies param description. Above we don't provide any description and only use type, which, as you're rightly point out, is weird. How about I just sort it out and have both `:param` and `:type` :)

Reply:
SGTM